### PR TITLE
Add emoji-material module for material bindings.

### PR DESCRIPTION
**Pull request by vanniktech**
First step. LayoutFactory will follow.

### CODE REVIEW

File: emoji/src/main/java/com/vanniktech/emoji/EmojiManager.java [hunk 1]
```diff
@@ -216,7 +216,7 @@ Pattern getEmojiRepetitivePattern() {
     return emojiMap.get(candidate.toString());
   }
 
-  void verifyInstalled() {
+  public void verifyInstalled() {
```
Comment (vanniktech) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiManager.java]: Not happy about this but required :(

File: build.gradle [hunk 2]
```diff
@@ -112,4 +113,11 @@ wrapper {
...
   task allDependencies(type: DependencyReportTask) {}
-}
+
+  project.plugins.withId("com.android.library") {
+    // TODO replace with https://issuetracker.google.com/issues/72050365 once released.
+    project.android.libraryVariants.all {
+      it.generateBuildConfigProvider.configure { it.enabled = false }
+    }
+  }
+}
```
Comment (mario) [inline build.gradle]: New line.

File: settings.gradle [hunk 3]
```diff
@@ -1 +1,8 @@
-include ':app', ':emoji', ':emoji-ios', ':emoji-google', ':emoji-twitter', ':emoji-google-compat', ':emoji-kotlin'
+include ':app'
+include ':emoji'
+include ':emoji-google'
+include ':emoji-google-compat'
+include ':emoji-ios'
+include ':emoji-kotlin'
+include ':emoji-material'
+include ':emoji-twitter'
```
Comment (mario) [inline settings.gradle]: New line?

**Comment by mario** [comment]
Looks decent, but are we copy/pasting too much code between material and regular widgets? Can that be unified?

**Comment by vanniktech** [comment]
> Looks decent, but are we copy/pasting too much code between material and regular widgets? Can that be unified?

I've tried 1-2 years ago and I gave up. Can try again but can be a follow up PR.

**Comment by mario** [comment]
Never give up :P

Needs rebase, then I can approve.

**Comment by vanniktech** [comment]
Done so

**Comment by vanniktech** [comment]
https://github.com/vanniktech/Emoji/issues/398

**Comment by vanniktech** [comment]
Waiting for @rubengees's code review

**Pull request merged**